### PR TITLE
Add README and use protobuf for trie

### PR DESCRIPTION
**Pull request by yhwang**
Add README to introduce the repo and change
the plain text trie file to binary by using
protobuf. It significantly reduces the file
size.

Signed-off-by: Yihong Wang <yh.wang@ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,66 @@
 [![Build Status](https://travis-ci.org/yhwang/ds2-tfjs.svg?branch=master)](https://travis-ci.org/yhwang/ds2-tfjs)
 
 # Enable the DeepSpeech2 in a browser
+
+This repo provides a web application that uses the tensorflow.js to serve the pre-trained DeepSpeech2 model
+in a browser. It processes the audio input and feeds the audio spectrogram into the model. It also contains
+the CTCDecoder implemetation to process the output of the DeepSpeech2 model, including character level and
```
Comment (tedhtchang) [inline README.md]: implemetation -> implementation

File: README.md [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+# Assuming you are at the repository's root directory
+mkdir -p public/model
+
+# Change the model.tar.gz to where your download file is
+tar xf model.tar.gz -C public/model
+```
+
+## N-Gram language model and Trie
+ 
+In order to reduce the network overhead, we choosed [3-gram ARPA LM and pruned with theshold 1e-7](http://www.openslr.org/resources/11/3-gram.pruned.1e-7.arpa.gz).
```
Comment (tedhtchang) [inline README.md]: choosed -> chose

File: README.md [hunk 3]
```diff
@@ -1,3 +1,66 @@
...
+Its binary format is about 26MB. You can download it from [here](https://ibm.box.com/v/3-gram-binary).
+(You need to open the link and click the `Download` button to dowload the file.)
```
Comment (tedhtchang) [inline README.md]: dowload -> download

File: README.md [hunk 4]
```diff
@@ -1,3 +1,66 @@
...
+Then extract it to `public` directory. For example:
+```
+# Change the 3-gram.binary.tar.gz to where your download file is
+ tar xf 3-gram.binary.tar.gz -C public
+```
+
+And trie file is [here](https://ibm.box.com/v/3-gram-trie-binary). 
+(You need to open the link and click the `Download` button to dowload the file.)
```
Comment (tedhtchang) [inline README.md]: dowload -> download

File: README.md (hunk 1 above)
Comment (yhwang) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (yhwang) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (yhwang) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (yhwang) [reply README.md]: fixed

**Pull request merged**